util/string_util.h

```diff
@@ -16,7 +16,7 @@ extern std::vector<std::string> StringSplit(const std::string& arg, char delim);
 
 template <typename T>
 inline std::string ToString(T value) {
-#if !(defined OS_ANDROID) && !(defined CYGWIN)
+#if !(defined OS_ANDROID) && !(defined CYGWIN) && !(defined __FreeBSD__)
```

Comment:
On 28-6-2016 18:50, Igor Canadi wrote:
> In util/string_util.h
> <https://github.com/facebook/rocksdb/pull/1187#discussion_r68796141>:
> 
>> @@ -16,7 +16,7 @@ extern std::vector<std::string> StringSplit(const std::string& arg, char delim);
>>  
>>  template <typename T>
>>  inline std::string ToString(T value) {
>> -#if !(defined OS_ANDROID) && !(defined CYGWIN)
>> +#if !(defined OS_ANDROID) && !(defined CYGWIN) && !(defined __FreeBSD__)
> 
> Can you pls replace *FreeBSD* with OS_FREEBSD? We #define OS_FREEBSD in
> our detection of target OS:
> https://github.com/facebook/rocksdb/blob/master/build_tools/build_detect_platform#L138

Hi Igor,

I've done it the sort of blunt way.
Closed the previous pull, en created a new one: #1190

Thanx, for addressing this.